ui/app/pages/add-token/add-token.container.js

```diff
@@ -13,6 +14,7 @@ const mapStateToProps = (state) => {
     mostRecentOverviewPage: getMostRecentOverviewPage(state),
     tokens,
     pendingTokens,
+    showSearchTab: getIsMainOrTestNet(state),
```

Comment:
I don't think we want to show this on testnets either. These contracts don't exist on testnets.

Reply:
My rationale here was that the testnets should reflect as much of the mainnet as possible.

Reply:
The tokens in the list only have mainnet contract addresses. If any of them work on test nets its happen stance, not by design.

Reply:
Exactly - this isn't something we get a say in. We don't own these contracts, so we can't put them on testnets. They just aren't there.

If an e2e test relies upon this being here, we can check for `process.env.IN_TEST` to leave it there during e2e tests. We do that here and there, for cases where a thing only makes sense on Mainnet but we want some limited e2e test coverage anyway.